lib/rucio/core/replica.py

```diff
@@ -1584,6 +1586,12 @@ def delete_replicas(rse_id, files, ignore_availability=True, session=None):
                                                      RuleState.WAITING_APPROVAL))).\
             delete(synchronize_session=False)
 
+    # Remove DID Metadata
+    for chunk in chunks(deleted_did_meta, 100):
```

Comment:
Here something similar to 
https://github.com/rucio/rucio/blob/6189936fd27ada4c729b7cc15e061387ec7121cb/lib/rucio/core/did.py#L616
should be done.